appcelerator/titanium_mobile
# feat(android): update kotlin language support to 1.4.0

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28102

**Summary:**
Updates module builds to use kotlin language 1.4.0.
https://android-developers.googleblog.com/2020/08/new-language-features-and-more-in.html

**Test:**
1. Download `ti.kotlin.zip` attached to [TIMOB-25896](https://jira.appcelerator.org/browse/TIMOB-25896).
2. Unzip it.
3. Copy the below `TiKotlin14.kt` file to folder: `./ti.kotlin/android/src/ti/kotlin`
4. `CD` to directory: `./ti.kotlin/android`
5. Run: `appc run -p android`
6. Verify app builds and runs successfully.

_./ti.kotlin/android/src/ti/kotlin/TiKotlin14.kt_
```kt
package ti.kotlin;

class TiKotlin14
{
	fun test()
	{
		val colorList = listOf(
			"red",
			"green",
			"blue",  // Trailing comma support requires kotlin 1.4
		);
	}
}
```
